uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1021.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+trait CRM_HRCore_Upgrader_Steps_1021 {
+
+  /**
+   * Creates the Missing Log Tables after CiviHR Upgrade
+   */
+  public function upgrade_1021() {
+    $this->up1021_fixMissingLogTables();
+
+    return TRUE;
+  }
+
+  /**
+   * Creates the Missing Log Tables After Drupal upgrade
```

Comment:
Is this CiviHR or Drupal upgrade? The previous comment saves CiviHR Upgrade.

Reply:
Actually is CiviCRM Upgrade. Fixed